modules/test/playwright/tests/account-admin-web/accounts.spec.ts

```diff
@@ -189,3 +190,57 @@ test('LPD-18484 Add account website', async ({
 		page.getByRole('cell', {name: 'https://www.website.com'})
 	).toBeVisible();
 });
+
+test('LPD-28161 Can view role and organizations printed as script', async ({
```

Comment:
please rename the description like this "Can view role and organization name escaped"